.github/actions/install-dashboards/action.yml

```diff
@@ -54,32 +62,65 @@ runs:
+      if: ${{ runner.os == 'Windows' }}
+      run: |
+        curl -SLO https://ci.opensearch.org/ci/dbc/distribution-build-opensearch-dashboards/${{ env.OPENSEARCH_VERSION }}/latest/windows/x64/zip/dist/opensearch-dashboards/opensearch-dashboards-${{ env.OPENSEARCH_VERSION }}-windows-x64.zip
+        unzip -q opensearch-dashboards-${{ env.OPENSEARCH_VERSION }}-windows-x64.zip
+      shell: bash
 
+    - name: Get and Setup OpenSearch-Dashboards Linux
+      if: ${{ runner.os == 'Linux' }}
+      run: |
+        wget https://ci.opensearch.org/ci/dbc/distribution-build-opensearch-dashboards/${{ env.OPENSEARCH_VERSION }}/latest/linux/x64/tar/dist/opensearch-dashboards/opensearch-dashboards-${{ env.OPENSEARCH_VERSION }}-linux-x64.tar.gz
+        tar -xzf opensearch-dashboards-${{ env.OPENSEARCH_VERSION }}-linux-x64.tar.gz
+      shell: bash
+    
     - id: tool-versions
       run: |
         echo "node_version=$(cat .node-version)" >> $GITHUB_OUTPUT
         echo "yarn_version=$(jq -r '.engines.yarn' package.json)" >> $GITHUB_OUTPUT
       working-directory: OpenSearch-Dashboards
       shell: bash
 
-    - uses: actions/setup-node@v1
+    - name: Setup node
+      uses: actions/setup-node@v1
       with:
-        node-version: ${{ steps.tool-versions.outputs.node_version }}
+        node-version: ${{ env.node_version }}
         registry-url: 'https://registry.npmjs.org'
 
-    - name: Setup Opensearch Dashboards
-      run: |
-        npm uninstall -g yarn
-        echo "Installing yarn ${{ steps.tool-versions.outputs.yarn_version }}"
-        npm i -g yarn@${{ steps.tool-versions.outputs.yarn_version }}
-        yarn cache clean
-        yarn add sha.js
-      working-directory: OpenSearch-Dashboards
-      shell: bash
-
     - name: Bootstrap the OpenSearch Dashboard
       uses: nick-fields/retry@v2
       with:
         timeout_minutes: 20
         max_attempts: 2
         command: yarn --cwd OpenSearch-Dashboards osd bootstrap --oss
+
+    - id: build-plugin
+      run: |
+        yarn build
+      working-directory: ${{ steps.determine-plugin-directory.outputs.plugin-directory }}
+      shell: bash
+
+    - id: where-are-we
+      run: |
+        ls
+      shell: bash
+    
+    - name: Remove Security Plugin
+      if: ${{ runner.os == 'Windows' || runner.os == 'Linux' }}
+      continue-on-error: true
+      run: |
+        echo "Remove Dashboards Security if Present"
+        opensearch-dashboards-${{ env.OPENSEARCH_VERSION}}/bin/opensearch-dashboards-plugin remove securityDashboards 
+      shell: bash
+
+    - name: Install Security Plugin 
+      if: ${{ runner.os == 'Windows' || runner.os == 'Linux' }}
+      run: |
+        echo "Install Dashboards Security from Build"
+        opensearch-dashboards-${{ env.OPENSEARCH_VERSION}}/bin/opensearch-dashboards-plugin install file:$PWD/OpenSearch-Dashboards/plugins/security-dashboards-plugin/build/security-dashboards-${{ env.PLUGIN_VERSION}}.zip 
+      shell: bash
```

Comment:
Is the `shell: bash` also works on Windows runner?

Reply:
Yea apparently it does, I was surprised to learn this as well.